Tactics/Sym/AxEffects.lean

```diff
@@ -78,6 +78,10 @@ structure AxEffects where
   However, if SP is written to, no effort is made to prove alignment of the
   new value; the field will be set to `none` -/
   stackAlignmentProof? : Option Expr
+
+  /-- `sideContitions` are proof obligations that come up during effect
```

Comment:
```suggestion
  /-- `sideConditions` are proof obligations that come up during effect
```

Reply:
These side conditions only stem from the SP alignment proof. That deserves a mention here.

Reply:
I've expanded the docstring